bin/yb-ctl

```diff
@@ -1213,8 +1213,8 @@ class ClusterControl:
         if not self.options.is_shell_master:
             command_list += ["--master_addresses {}".format(self.options.master_addresses)]
 
-        if self.options.enable_ysql:
-            command_list.append("--use_initial_sys_catalog_snapshot")
+        if not self.options.enable_ysql:
+            command_list.append("--enable_ysql=false")
```

Comment:
Given that this commit, and YB side change could land as independent steps, be explicit in the if and else cases.

So, prefer something like:

if self.options.enable_ysql:
   command_list.append("--enable_ysql=true")
else 
   command_list.append("--enable_ysql=false")